docs/docs/running-hello-world.md

```diff
@@ -1,68 +1,80 @@
-# Compiling and Running your First Enclave
+# Compiling and running your first enclave
 
 ## Introduction
 
-This tutorial describes how to compile and run the sample hello world app, which you can find in the 
-[Conclave tutorials repository](https://github.com/R3Conclave/conclave-tutorials/tree/HEAD/hello-world). This app contains the `ReverseEnclave`, 
-which reverses a string provided by the client and passes it back.
+This tutorial describes how to compile and run the
+[sample hello world app](https://github.com/R3Conclave/conclave-tutorials/tree/HEAD/hello-world).
+This application contains an enclave `ReverseEnclave`, which reverses a string provided by the client and passes it 
```

Comment:
Remove "ReverseEnclave". So it would read "This application contains an enclave which reverses ..."